README.md

```diff
@@ -1,16 +1,16 @@
-# Stratos UI
+# Stratos
 
-[![Build Status](https://travis-ci.org/SUSE/stratos-ui.svg?branch=master)](https://travis-ci.org/SUSE/stratos-ui)
-[![GitHub release](https://img.shields.io/github/release/SUSE/stratos-ui.svg)](https://github.com/SUSE/stratos-ui/releases/latest)
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/SUSE/stratos-ui/blob/master/LICENSE.md)
+[![Build Status](https://travis-ci.org/cloudfoundry-incubator/stratos.svg?branch=master)](https://travis-ci.org/cloudfoundry-incubatorSUSE/stratos)
+[![GitHub release](https://img.shields.io/github/release/cloudfoundry-incubator/stratos.svg)](https://github.com/cloudfoundry-incubator/stratos/releases/latest)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/cloudfoundry-incubator/stratos/blob/master/LICENSE.md)
```

Comment:
License link doesn't work. LICENSE and NOTICE seem to be checked in master with the extension `.md`